CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2444,12 +2498,21 @@ private void OnMouseButton(MouseButtonEventArgs e)
                     //Anything greater than 3 then we send click count of 1
                     var clickCount = e.ClickCount;
 
-                    if(clickCount > 3)
+                    if (clickCount > 3)
                     {
                         clickCount = 1;
                     }
 
-                    browser.GetHost().SendMouseClickEvent((int)point.X, (int)point.Y, (MouseButtonType)e.ChangedButton, mouseUp, clickCount, modifiers);
+                    if (mouseTeleport.IsInsideOriginalRect((int)point.X, (int)point.Y))
```

Comment:
Under what instance do we want to ignore the mouse click?

These methods should simple call `GetAdjustedMouseCoords`, no logic should be in any of these methods.

Reply:
I moved the logic to the class.